openslides/motions/static/js/motions/site.js

```diff
@@ -21,7 +21,7 @@ angular.module('OpenSlidesApp.motions.site', ['OpenSlidesApp.motions', 'OpenSlid
                 * other project that might want to use this HTML to PDF parser.
                 * https://github.com/OpenSlides/OpenSlides/issues/2361
                 */
-                return converter.convertHTML($scope.lineBrokenText, $scope);
+                return converter.convertHTML(motion.getTextWithLineBreaks($scope.version, 0), $scope);
```

Comment:
@tsiegleauq You could maybe just test if you can pass undefined. I don't know how the software behaves, but you may just write a check for this [here](https://github.com/OpenSlides/OpenSlides/blob/master/openslides/motions/static/js/motions/linenumbering.js#L89) where the highlighting is applied. Something like:
```
if (typeof highlight === 'undefined') {
highlight=0;
}
```